apps/builder/app/builder/features/style-panel/sections/box-shadows/box-shadow-layer.tsx

```diff
@@ -18,8 +18,12 @@ import { toValue } from "@webstudio-is/css-engine";
 import { ColorThumb } from "../../shared/color-thumb";
 import type { LayerProps } from "../../style-layers-list";
 
-const useLayer = (layer: TupleValue) => {
+const useLayer = (layer: TupleValue | FunctionValue) => {
```

Comment:
Why Both supported? Imo better to always have strong type for Layer even if there is just 1 function